src/components/folder/ContantList.tsx

```diff
@@ -0,0 +1,27 @@
+import PostCard from './PostCard';
+import { EmptyBox, PostCardWrap } from '../../pages/folder/folderStyle';
+import { IFolderContent } from '../../pages/folder/interface';
+
+interface IFolderList {
+  contant: IFolderContent[] | [] | undefined | null;
+  loading: boolean;
```

Comment:
IFolderContent[] 타입 내부에 [] 도 들어있어서 하나만 적어주시면 될 거 같아요!

```suggestion
  contant: IFolderContent[] | undefined | null;
  isLoading: boolean;
```